src/Umbraco.Web/Mvc/MinifyJavaScriptResultAttribute.cs

```diff
@@ -24,7 +38,7 @@ public override void OnResultExecuting(ResultExecutingContext filterContext)
             if (filterContext.Result == null) return;
             var jsResult = filterContext.Result as JavaScriptResult;
             if (jsResult == null) return;
-            if (filterContext.HttpContext.IsDebuggingEnabled) return;
+            if (_hostingEnvironment.IsDebugMode) return;
```

Comment:
I assume this change is really just pre-emptively knowing that these APIs will be different in netcore MVC?

Reply:
I know these will change, I just try to align and abstract our direct usage of `HttpContext`